lib/read/lib/Cardano/Wallet/Read/Block/Gen/Build.hs

```diff
@@ -169,7 +169,7 @@ data ChainBuild m (address :: Type) a where
     Tx :: TxF m () -> ChainBuild m addr TxId
     TxP :: TxParameters -> ChainBuild m addr TxId
     NewAddress :: addr -> ChainBuild m addr Address
-    Block :: EraFunSel era -> SlotNo -> ChainBuild m addr ()
+    Block :: IsEra era => Proxy era -> SlotNo -> ChainBuild m addr ()
```

Comment:
```suggestion
    Block :: IsEra era => Era era -> SlotNo -> ChainBuild m addr ()
```

Using the singleton type would be even better, but `Proxy` is perfectly fine here.